rosidl_typesupport_introspection_c/rosidl_typesupport_introspection_c/cli.py

```diff
@@ -25,6 +25,7 @@
 
 
 class GenerateIntrospectionCTypesupport(GenerateCommandExtension):
+    """Generator class for generating the C implementation of the type support."""
```

Comment:
Imperative mood: `Generate the C type support implementation.`

Reply:
right, 8471f31db5b26c0b758688c6c28d16b9baae5e53